test/test.js

```diff
@@ -7,6 +7,14 @@ describe('resolve-dependency-path', function() {
     assert(resolved.indexOf(__dirname) === 0);
   });
 
+  it('resolves with multiple periods in the dependency path', function() {
+    var depPath = './bar.baz.qux';
```

Comment:
Hmm, how does this reach the else? Doesn't this technically have the ".qux" extension?